### DISCUSSION TITLE
Form stuck with readonly inputs and disabled buttons when dispatching an event to another component

### DISCUSSION THREAD

**Discussion by gbalduzzi** [livewire/livewire#8194] [7. Bugs (archived)]
### Livewire version

v3.4.6

### Laravel version

v10.45.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a livewire component with a form (including an input and a button).
2. In the wire:submit form handler dispatch an event:  `wire:submit="$dispatch('some-event')"`
3. Create a parent component, with an event handler for `some-event`
4. In the parent component, include the child component with the form
5. Submit the form
6. The form will be stuck with the input as `readonly` and the button as `disabled`

### Repository or Wirebox URL

https://wirebox.app/b/4kowd

### Additional information

The event handler in the parent component works correctly

Adding a fake event handler in the child component solved the issue:

```php
#[On('some-event')]
public function submit(array $filters): void
{
    // do nothing
}
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@gbalduzzi thanks for reporting! Definitely looks like a bug to me. Can you submit a failing test PR? Thanks!

**Reply by danie-ramdhani** [reply]
@joshhanley i got the failing test, but i don't know where i should put it (or should i make new one named SupportMagicActions). 
This related with magic actions (`wire:submit="$parent..."` or in this discussion is using `wire:submit="$dispatch..."`)

**Reply by joshhanley** [reply]
@danie-ramdhani thanks! I would probably put it in `SupportNestingComponents`.

**Reply by danie-ramdhani** [reply]
Added failing test https://github.com/livewire/livewire/pull/8241

**Reply by danie-ramdhani** [reply]
merged with bug fix (idk about the `$dispatch` action because i honestly can't find a case submitting form using dispatch without sending the form data but it fix when using `$parent` action)

**Reply by gbalduzzi** [reply]
Thank you @danie-ramdhani !
